### PR TITLE
ScholarX featured stories cards fix 

### LINKED ISSUE #867: Change card view in scholarx
**Describe the bug**
In the Scholarx page featured stories section, there are a set of cards. Those are different from the home page cards. 

**To Reproduce**
1. Go to `ScholarX page` (from navigation-> projects-> ScholarX)
2. Scroll down to `Featured Stories` section
3. Got to `Home page`
4. Scroll down to `Our Projects` section
5. See the difference

**Expected behavior**
The `Scholarx page`,` Featured Stories` cards need to be similar(with curved edges) to the `Home page` `Our Projects` cards. 
The images on the featured stories cards need to be fitted into the new curved cards.

### PR DESCRIPTION
**Pull request by akeel230**
## Purpose
fix #867 

## Goals
make ScholarX page featured story cards look similar to Home page cards.

## Approach

1. Inspect the Home page , copy the cards settings , replace with the feature page cards
2. insert copied info from featured stories
3. Make the border round by styling
4. Fix up the padding to remove borders

### Preview Link

https://pr-1020-sef-site.surge.sh/

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

## Related PRs
none

##Screenshots
Screenshot (153)
Screenshot (152)

## Test environment
http://127.0.0.1:9000/scholarx/2021/ 

## Learning

- Removing and adding padding options to cards
- Removing borders

### CODE REVIEW

**Comment by akeel230** [comment]
@Gravewalker666 Sorry i mistakenly deleted the last repo i forked so had to make a new repo while working so the changes could not be pushed into the last PR had to create a new one.

**Review by janethavi — CHANGES_REQUESTED** [review]
Great work @akeel230 You have done a really good job. A small thing, there is a border around the card. Is there any way of removing it?

**Comment by akeel230** [comment]
@janethavi alright thanks for the feedback, ill see what I can do for the border

**Comment by akeel230** [comment]
@janethavi i removed the border , take a look .
Screenshot (155)

**Review by janethavi — DISMISSED** [review]

File: scholarx/2021/index.html [hunk 1]
```diff
@@ -655,30 +655,36 @@ <h5>Projects :</h5>
...
-                        </div>
-                        <div class="stats ml-auto">
-                            <i class="far fa-clock"></i> {{readingTime}} min read
+                
+          <div class="col-md-4" > 
+              <div class="card px-0 " style="border: none;">
+                  <a href="{{url}}" target="_blank" class="card shadow project-card">
+                     <div class="card-body">
+                        <img src="images/featured-stories/{{image}}" class="img-fluid  img-full" alt="OneLive">
+                        <p class="card-text text-darker text-left" style="padding-left: 5%;">
```
Comment (Gravewalker666) [inline scholarx/2021/index.html]: Let's not use inline CSS. Can you please make two separate CSS classes? Or you can find an existing class that does the same 
BTW it looks awesome now, good job

**Review by Gravewalker666 — CHANGES_REQUESTED** [review]

**Review by Gravewalker666 — CHANGES_REQUESTED** [review]
image
The border radius of the card and the image is not the same. Can you fix that too please??

**Comment by akeel230** [comment]
@Gravewalker666 I removed the inline style and replaced in in the separate stylesheet and now the border of the image tallies with the border of the card.
Screenshot (164)

File: scholarx/2021/index.html (hunk 1 above)
Comment (akeel230) [reply scholarx/2021/index.html]: Done

Comment (Gravewalker666) [reply scholarx/2021/index.html]: @akeel230 you have missed one in the line 668

File: scholarx/2021/index.html [hunk 2]
```diff
@@ -655,30 +655,36 @@ <h5>Projects :</h5>
...
-                            <i class="far fa-clock"></i> {{readingTime}} min read
+                
+          <div class="col-md-4" > 
+              <div class="card px-0 ">
+                  <a href="{{url}}" target="_blank" class="card shadow project-card">
+                     <div class="card-body">
+                        <img src="images/featured-stories/{{image}}" class="img-fluid  img-full" alt="OneLive">
+                        <p class="card-text text-darker text-left" style="padding-left: 5%;">
+                            {{title}}
+                        </p>
```
Comment (Gravewalker666) [inline scholarx/2021/index.html]: for this you can just use something like `ml-<number>` from bootstrap itself

File: scholarx/2021/index.html (hunk 1 above)
Comment (akeel230) [reply scholarx/2021/index.html]: sorry, My bad ill fix that

File: scholarx/2021/index.html (hunk 2 above)
Comment (akeel230) [reply scholarx/2021/index.html]: Thank You for the guideline, i used it lemme update and push

**Comment by akeel230** [comment]
@Gravewalker666 Please take a look now . I've made the changes you requested.

**Review by Gravewalker666 — APPROVED** [review]
Great work @akeel230 
Very professional 👏

**Pull request merged**